website/docs/cloud-docs/run/states.mdx

```diff
@@ -22,19 +22,50 @@ _Leaving this stage:_
 - If the user discards the run before it starts, the run does not continue (**Discarded** state).
 - If the run is first in the queue, it proceeds automatically to the plan stage (**Planning** state).
 
-## 2. The Plan Stage
+## 2. The Fetching Stage
 
-A run goes through different steps during the plan stage depending on whether or not Terraform Cloud needs to fetch the configuration from VCS. Terraform Cloud automatically archives configuration versions created through VCS when all runs are complete and then re-fetches the files for subsequent runs.
+Terraform Cloud may need to fetch the configuration from VCS prior to starting the plan. Terraform Cloud automatically archives configuration versions created through VCS when all runs are complete and then re-fetches the files for subsequent runs.
 
 _States in this stage:_
 
 - **Fetching:** If Terraform Cloud has not yet fetched the configuration from VCS, the run will go into this state until the configuration is available.
+
+_Leaving this stage:_
+
+- If Terraform Cloud encounters an error when fetching the configuration from VCS, the run does not continue (**Plan Errored** state).
+- If Terraform successfully fetches the configuration, the run moves to the next stage.
+
+## 3. The Pre-Plan Stage
+
+The pre-plan phase only occurs if there are enabled [run tasks](/cloud-docs/workspaces/settings/run-tasks) in the workspace that are configured to begin before Terraform creates the plan. Terraform Cloud sends information about the run to the configured external system and waits for a `passed` or `failed` response to determine whether the run can continue. The information sent to the external system includes the configuration version of the run.
+
+All runs can enter this phase, including [speculative plans](/cloud-docs/run/#speculative-plans). 
+
+
+_States in this stage:_
+
+- **Pre-plan running:** Terraform Cloud is waiting for a response from the configured external system(s).
+ 
+ External systems must respond initially with a `200 OK` acknowledging the request is in progress. After that, they have 10 minutes to return a status of `passed`, `running`, or `failed`. If the timeout expires, Terraform Cloud assumes that the run tasks is in the `failed` status.
```

Comment:
Are we going to repeat ourselves a few times on this page?

Reply:
Also, the other time this is mentioned seems to be a unordered list:
https://github.com/hashicorp/terraform-docs-common/pull/72/files#diff-9c4b29067fbb525b780309e425de54e31d50d132eb3dcd46ca739a137277b057R90

Reply:
I've fixed this up to be consistent (unorderd list)

However I have no answer to the repeating bit.  It may be a necessary evil.